### ISSUE TITLE
pipewire does not work after suspend

### ISSUE THREAD

**Issue by notthewave** [void-linux/void-packages#35206]
### System

* xuname:  
  *Void 5.15.16_1 x86_64-musl GenuineIntel uptodate rrrnDFFFFFFFFFFFFFFFFFFFF*
* package:  
  *pipewire-0.3.43_1*

### Expected behavior
Audio works after suspend

### Actual behavior
Audio does not work after suspend

### Steps to reproduce the behavior
do loginctl suspend
or zzz

and wake from sleep with pipewire

**Comment by Duncaen** [comment]
This is really not enough information, works for everyone else.

**Comment by paper42** [comment]
For getting more information, you run pipewire (and pipewire-pulse) in a terminal, suspend, wake the computer back up and post the output. I can not reproduce this.

**Comment by Duncaen** [comment]
Also check dmesg, I doubt pipewire itself has to or does anything special about waking up from suspend, probably more kernel related.

**Comment by notthewave** [comment]
The only thing Pipewire reports after suspend is:
```
[W][27921.874134] spa.bluez5   | [   bluez5-dbus.c: 1352 device_update_props()] unknown adapter /org/bluez/hci0
```
dmesg reports:
```
pipewire-media-[28789]: segfault at 44 ip 00007fbe3e59590c sp 00007ffdc6c4c3f0 error 4 in libspa-bluez5.so[7fbe3e56d000+65000]
```

**Comment by ar-jan** [comment]
Recently I'm seeing exactly the same after suspend: the playback device isn't listed in system settings anymore, and dmesg shows a segfault for pipewire media.

I killed all pipewire processes with `pkill -f "pipewire.*" -9`. Then:

`pipewire &`
`pipewire-pulse &`
This works to get audio playback back, and latter outputs this:
[W][41133.975939] mod.protocol-pulse | [        server.c:  573 start_unix_server()] server 0x55a2ef6c50f0: unlinking stale socket '/run/user/1000/pulse/native'

**Comment by notthewave** [comment]
https://gitlab.freedesktop.org/pipewire/pipewire/-/issues/2010 next version sopposedly fixes this

**Comment by notthewave** [comment]
new release is out

**Comment by st3r4g** [comment]
waiting for https://gitlab.freedesktop.org/pipewire/pipewire/-/issues/2069 to be sorted out

**Comment by tornaria** [comment]
I have the same issue. After suspend pipewire is dead. Just restarting pipewire solves it (no need to restart pipewire-pulse).

**Comment by ar-jan** [comment]
Next release landed https://gitlab.freedesktop.org/pipewire/pipewire/-/releases/0.3.45

**Comment by hervyqa** [comment]
> Next release landed https://gitlab.freedesktop.org/pipewire/pipewire/-/releases/0.3.45

Merged. https://github.com/void-linux/void-packages/pull/35300

**Comment by notthewave** [comment]
closing as this was fixed by 8c744e3c91e199030c0d39dec06951f82e658f2d
